conf-gmp-powm-sec failing to build on mac

Attempting to install cryptokit (OCaml 4.06.0) on my macOS 10.13.1 system leads to conf-gmp-powm-sec trying to install. It fails with:

    + cc -c -I/usr/local/include test.c
    test.c:1:10: fatal error: 'gmp.h' file not found
    #include <gmp.h>
             ^~~~~~~
    1 error generated.

Any ideas?

--- a/packages/conf-gmp-powm-sec/conf-gmp-powm-sec.1/opam
+++ b/packages/conf-gmp-powm-sec/conf-gmp-powm-sec.1/opam
@@ -6,7 +6,8 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 dev-repo: "https://github.com/ocaml/opam-repository.git"
 license: "GPL"
 build: [
-  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"]
+  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "darwin"}
+  ["sh" "-exc" "cc -c $CFLAGS -I/opt/local/include -I/usr/local/include test.c"] {os = "darwin"}
 ]
 depends: [
     "conf-gmp"

